src/test/test-path.c

```diff
@@ -79,13 +79,10 @@ static Service *service_for_path(Manager *m, Path *path, const char *service_nam
 }
 
 static void check_states(Manager *m, Path *path, Service *service, PathState path_state, ServiceState service_state) {
-        usec_t ts;
-        usec_t timeout = 2 * USEC_PER_SEC;
-
         assert_se(m);
         assert_se(service);
 
-        ts = now(CLOCK_MONOTONIC);
+        usec_t ts = now(CLOCK_MONOTONIC);
```

Comment:
if you change this around I'd always calculate the end point here, to separate this out. i.e. 

```c
usec_t end = now(COCK_MONOTONIC) + 30 * USER_PER_SEC;
```

and then further down just `if (n >= end) … `

Reply:
Yes, that's much better.